src/main/features/claimant-response/routes/payment-option.ts

```diff
@@ -61,12 +61,17 @@ export class PaymentOptionPage extends AbstractPaymentOptionPage<DraftClaimantRe
           paymentLength: paymentPlanConvertedToDefendantFrequency.calculatePaymentLength()
         }
         return courtOfferedPaymentIntention
-      }
-
-      if (claimResponse.paymentIntention.paymentOption === PaymentOption.BY_SPECIFIED_DATE) {
-        courtOfferedPaymentIntention.paymentDate = paymentPlanFromDefendantFinancialStatement.calculateLastPaymentDate()
-        courtOfferedPaymentIntention.paymentOption = PaymentOption.BY_SPECIFIED_DATE
+      } else {
+        const paymentPlanConvertedToMonthlyFrequency: PaymentPlan = paymentPlanFromDefendantFinancialStatement.convertTo(Frequency.MONTHLY)
 
+        courtOfferedPaymentIntention.paymentOption = PaymentOption.INSTALMENTS
+        courtOfferedPaymentIntention.repaymentPlan = {
+          firstPaymentDate: paymentPlanConvertedToMonthlyFrequency.startDate,
+          instalmentAmount: Math.round(paymentPlanConvertedToMonthlyFrequency.instalmentAmount * 100) / 100,
```

Comment:
the `instalmentAmount * 100) / 100` is used in 3 places in this file. can you please pull it out to a function now and add some tests for it?

Reply:
i have updated it to use lodash round function. Don't think we need test now!

if you agree, i will create seperate PR to update to lodash round function in other court determination calculations...